app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,44 @@
+footer {
+  background-color: rgb(232, 230, 230);
+  width: 95%;
+  margin: 0 auto;
+  height: 200px;
+  border-radius: 15px;
+
+    a {
+      color: #8f8f8f;
+
+      &:hover {
+        color: #55679b;
+      }
+    }
+  .wrapper {
+    display: flex;
+    align-items: center;
+    padding: 40px 0 40px 0;
+
+    .col-1 {
+      margin: 0 auto;
+      flex-grow: 1;
```

Comment:
col-1 is really not an explicit name. Can you rename that to something that more understandable ?